ndmg/stats/qa_reg.py

```diff
@@ -56,17 +56,16 @@ def reg_mri_pngs(
     else:  # dim=3
         mr_data = mri_data
 
-    cmap1 = LinearSegmentedColormap.from_list("mycmap1", ["black", "magenta"])
-    cmap2 = LinearSegmentedColormap.from_list("mycmap2", ["black", "green"])
+    cmap1 = LinearSegmentedColormap.from_list("mycmap1", ["white", "magenta"])
+    cmap2 = LinearSegmentedColormap.from_list("mycmap2", ["white", "green"])
```

Comment:
fine, although a lot of this code is being repeated across PRs. Like I said in slack, consolidate any repeated code into one place (maybe through a group-PR).